src/commands/env/delete/sandbox.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Flags } from '@oclif/core';
+import { Messages, Org } from '@salesforce/core';
+import { SfCommand } from '@salesforce/sf-plugins-core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-env', 'delete_sandbox');
+
+export interface SandboxDeleteResponse {
+  orgId: string;
+  username: string;
+}
+export default class EnvDeleteSandbox extends SfCommand<SandboxDeleteResponse> {
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessages('examples');
+  public static flags = {
+    'target-env': Flags.string({
```

Comment:
This should be `target-org` if you will only ever pass orgs to this command as specified in the [styleguide](https://salesforce.quip.com/NJcjAyeE37gX#cSWACA91Alf).
That's what the [deploy/retrieve metadata commands do](https://github.com/salesforcecli/plugin-deploy-retrieve-metadata/blob/74affeb5dc095286adb6f54d00c0611a59fe0f23/src/commands/deploy/metadata.ts#L66) too.